src/Queries/Delete.php

```diff
@@ -5,6 +5,15 @@
 
 use SimpleCrud\Table;
 
+/**
+ * @method void setFlag(string $flag, bool $enable = true)
+ * @method self where(string $condition, ...$bindInline)
+ * @method self orWhere(string $condition, ...$bindInline)
+ * @method self catWhere(string $condition, ...$bindInline)
+ * @method self orderBy(string $expr, string ...$exprs)
+ * @method self limit(int $limit)
+ * @method self offset(int $offset)
+ */
```

Comment:
Not sure if it's a good idea because require more maintenance work, not only for this Query but all other Queries.

Reply:
The second way to resolve this is write a PHPStan extension.